install/samples/lazyload/slimeboot_cluster_prometheus.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: config.netease.com/v1alpha1
+kind: SlimeBoot
+metadata:
+  name: lazyload
+  namespace: mesh-operator
+spec:
+  image:
+    pullPolicy: Always
+    repository: docker.io/slimeio/slime-lazyload
+    tag: master-11af379-dirty_ccc2a74
+  module:
+    - name: lazyload
+      enable: true
+      fence:
+        wormholePort: # replace to your application service ports, and extend the list in case of multi ports
+          - "80"
+          - "9080"
+        namespace: # replace to your service's namespace which will use lazyload, and extend the list in case of multi namespaces
+          - default
+      global:
+        misc:
+          global_sidecar_mode: cluster # inform the mode of global-sidecar
+      metric: # indicate the metric source
+        prometheus:
+          address: http://prometheus.istio-system:9090
+          handlers:
+            destination:
+              query: |
+                sum(istio_requests_total{source_app="$source_app",reporter="destination"})by(destination_service)
+              type: Group
+  component:
+    globalSidecar:
+      enable: true
+      sidecarInject:
+        enable: true # should be true
+        # mode definition:
+        # "pod": sidecar auto-inject on pod level, need provide labels for injection
+        # "namespace": sidecar auto-inject on namespace level, no need to provide labels for injection
+        # if global_sidecar_mode is cluster, global-sidecar will be deployed in slime namespace, which does not enable auto-inject on namespace level, mode can only be "pod".
+        # if global_sidecar_mode is namespace, depending on the namespace definition, mode can be "pod" or "namespace".
+        mode: pod
+        labels: # optional, used for sidecarInject.mode = pod
+          sidecar.istio.io/inject: "true"
```

Comment:
more example for istio-rev scenary?

Reply:
add comment `# istio.io/rev: canary # use control plane revisions`